lib/common/util/config.js

```diff
@@ -121,7 +121,7 @@ function initializeConfig () {
   const languageId = window.localStorage.getItem('lang')
     ? window.localStorage.getItem('lang')
     : navigator.language
-  const active = languages.find(l => l._id === languageId) ||
+  const active = languages.find(l => languageId.includes(l._id)) ||
```

Comment:
I would say `l => l._id === languageId || languageId.startsWith(l._id)` to match the exact locale if it exists. 

But definitely at least: `l => languageId.startsWith(l._id)`.

Reply:
Definitely agree with this, updated to the first option in https://github.com/ibi-group/datatools-ui/pull/767/commits/a6e06b228d58580bd945bda5ec72adc12c5e0890